packages/@aws-cdk/aws-chatbot/test/slack-channel-configuration.test.ts

```diff
@@ -99,6 +99,34 @@ describe('SlackChannelConfiguration', () => {
     });
   });
 
+  test('adding topic after creating SlackChannel', () => {
+    const slackChannel = new chatbot.SlackChannelConfiguration(stack, 'MySlackChannel', {
+      slackWorkspaceId: 'ABC123',
+      slackChannelId: 'DEF456',
+      slackChannelConfigurationName: 'Test',
+    });
+
+    const topic = new sns.Topic(stack, 'MyTopic');
+    slackChannel.addNotificationTopic(topic);
+
+    expect(stack).toHaveResourceLike('AWS::Chatbot::SlackChannelConfiguration', {
+      ConfigurationName: 'Test',
+      IamRoleArn: {
+        'Fn::GetAtt': [
+          'MySlackChannelConfigurationRole1D3F23AE',
+          'Arn',
+        ],
+      },
+      SlackChannelId: 'DEF456',
+      SlackWorkspaceId: 'ABC123',
```

Comment:
Let's remove those properties from the assertion? I don't think this test is about verifying these.

Reply:
done